From f48e6a4ae948457e5b12f0f7dfc4f9547111d463 Mon Sep 17 00:00:00 2001
From: Tim Lunn <tim@feathertop.org>
Date: Sat, 15 Aug 2015 09:10:49 +1000
Subject: [PATCH] datetime: Add UTC+08:30 timezone for North Korea

These are symlinked to the previous image files until new ones are made

https://bugzilla.gnome.org/show_bug.cgi?id=753643
---
 panels/datetime/data/timezone_8.5.png     | 1 +
 panels/datetime/data/timezone_8.5_dim.png | 1 +
 panels/datetime/datetime.gresource.xml    | 2 ++
 3 files changed, 4 insertions(+)
 create mode 120000 panels/datetime/data/timezone_8.5.png
 create mode 120000 panels/datetime/data/timezone_8.5_dim.png

diff --git a/panels/datetime/data/timezone_8.5.png b/panels/datetime/data/timezone_8.5.png
new file mode 120000
index 0000000..e4d1020
--- /dev/null
+++ b/panels/datetime/data/timezone_8.5.png
@@ -0,0 +1 @@
+timezone_9.png
\ No newline at end of file
diff --git a/panels/datetime/data/timezone_8.5_dim.png b/panels/datetime/data/timezone_8.5_dim.png
new file mode 120000
index 0000000..5930745
--- /dev/null
+++ b/panels/datetime/data/timezone_8.5_dim.png
@@ -0,0 +1 @@
+timezone_9_dim.png
\ No newline at end of file
diff --git a/panels/datetime/datetime.gresource.xml b/panels/datetime/datetime.gresource.xml
index 3282dcd..ccbf955 100644
--- a/panels/datetime/datetime.gresource.xml
+++ b/panels/datetime/datetime.gresource.xml
@@ -81,6 +81,8 @@
     <file alias="timezone_-8_dim.png">data/timezone_-8_dim.png</file>
     <file alias="timezone_8.png">data/timezone_8.png</file>
     <file alias="timezone_8_dim.png">data/timezone_8_dim.png</file>
+    <file alias="timezone_8.5.png">data/timezone_8.5.png</file>
+    <file alias="timezone_8.5_dim.png">data/timezone_8.5_dim.png</file>
     <file alias="timezone_8.75.png">data/timezone_8.75.png</file>
     <file alias="timezone_8.75_dim.png">data/timezone_8.75_dim.png</file>
     <file alias="timezone_-9.png">data/timezone_-9.png</file>
-- 
2.5.0


